assisted_deployment.sh

```diff
@@ -68,11 +63,24 @@ spec:
 EOCR
 }
 
+function deploy_local_storage() {
+  oc adm new-project openshift-local-storage || true
```

Comment:
I think the `local-storage` operator will take care of this (since we are passing the target namespace)

Reply:
This code has just been moved around, but I'll test how it works without this piece

Reply:
Seems like that's not the case

```
Creating local storage operator group and subscription...
+(./assisted_deployment.sh:83): deploy_local_storage(): oc apply -f ocp/ostest/saved-assets/assisted-installer-manifests/01-local-storage-operator.yaml                                                                              
Error from server (NotFound): error when creating "ocp/ostest/saved-assets/assisted-installer-manifests/01-local-storage-operator.yaml": namespaces "openshift-local-storage" not found                                              
```

Reply:
> Seems like that's not the case
> 
> ```
> Creating local storage operator group and subscription...
> +(./assisted_deployment.sh:83): deploy_local_storage(): oc apply -f ocp/ostest/saved-assets/assisted-installer-manifests/01-local-storage-operator.yaml                                                                              
> Error from server (NotFound): error when creating "ocp/ostest/saved-assets/assisted-installer-manifests/01-local-storage-operator.yaml": namespaces "openshift-local-storage" not found                                              
> ```

Gotcha! thanks!

Let's add the debug/prints I mentioned and get this in then :smiley: